File: Luento3/W3/W3_T2/person.js
class Person {
    first_name;
    last_name;
    #age;

    constructor(fname, lname, age) {
        this.first_name = fname;
        this.last_name = lname;
        this.#age = age;
    }

    getAge() {
        return this.#age;
    }

    ageUp() {
        this.#age += 1;
    }

    getFullname() {
        return this.first_name + " " + this.last_name;
    }

    printFullname() {
        console.log(this.getFullname());
    }
}

module.exports = Person;

File: Luento3/W3/W3_T4/Person.java
public class Person {
    public String first_name;
    public String last_name;
    private int age;

    public Person(String fname, String lname, int age) {
        this.first_name = fname;
        this.last_name = lname;
        this.age = age;
    }

    public int getAge() {
        return age;
    }

    public void ageUp() {
        age += 1;
    }

    public String getFullname() {
        return first_name + " " + last_name;
    }

    public void printFullname() {
        System.out.println(getFullname());
    }
}
